### PR TITLE
Manual mounting overhaul

### PR DESCRIPTION
**Pull request by ImUrX**
Now it's based on a wheel that let's you choose the rotation

~~Blocked by SlimeVR/SolarXR-Protocol#114~~

### CODE REVIEW

**Comment by ImUrX** [comment]
no longer blocked ^^

**Comment by ImUrX** [comment]
I also block Reset mounting widget button if full reset status is currently on!

File: gui/public/i18n/en/translation.ftl [hunk 1]
```diff
@@ -145,9 +148,15 @@ tracker-table-column-url = URL
...
 tracker-rotation-front = Front
+tracker-rotation-front_left = Front-Left
+tracker-rotation-front_right = Front-Right
 tracker-rotation-left = Left
 tracker-rotation-right = Right
 tracker-rotation-back = Back
+tracker-rotation-back_left = Back-Left
+tracker-rotation-back_right = Back-Right
+tracker-rotation-custom = Custom
+tracker-rotation-overriden = (is getting overriden by mounting reset)
```
Comment (ButterscotchV) [inline gui/public/i18n/en/translation.ftl:159]: This is just a bit of a shorter way of saying the same thing :3
```suggestion
tracker-rotation-overriden = (overriden by mounting reset)
```

**Review by ButterscotchV — APPROVED** [review]

**Review by ButterscotchV — APPROVED** [review]

**Review by Louka3000 — APPROVED** [review]

**Comment by Louka3000** [comment]
image

**Review by Louka3000 — APPROVED** [review]
> It was never aligned
Vertically
Well it is
Idk if I can do css magic on this one

Problem for another PR I guess

**Pull request merged**